pages/user/signin.js

```diff
@@ -76,17 +76,12 @@ export default function Signin() {
         <br />
 
         <BtnSection className="redirection-btn">
-          <PostButton
-            name="signupbtn"
+          <btn.Blue
             buttonText="회원가입하기"
             onClickFunc={() => router.push('/user/signup')}
-          ></PostButton>
+          />
           <text>&nbsp;&nbsp;&nbsp;&nbsp;</text>
```

Comment:
```suggestion
          <div className="px=5"/>
```
https://tailwindcss.com/docs/padding

Reply:
반영완료